pkg/tasks/certs.go

```diff
@@ -31,3 +42,99 @@ func deployCAOnNode(s *state.State, node *kubeoneapi.HostConfig, conn ssh.Connec
 	s.Logger.Infoln("Uploading PKI files...")
 	return s.Configuration.UploadTo(conn, s.WorkDir)
 }
+
+func renewCerts(s *state.State) error {
+	if !s.ForceUpgrade {
+		s.Logger.Warn("Your control-plane certificates are about to expire in less then 90 days")
```

Comment:
Let's mention that you can use `--force-upgrade` to force renew certificates. That can be on a new line, i.e. another `s.Logger.Warn` call.

Reply:
done